Remotion/Web/Development.WebTesting/CoypuElementScopeFillInWithAndSendKeysExtensions.cs

```diff
@@ -105,11 +105,24 @@ private static void SetValueUsingSendKeys ([NotNull] this ElementScope scope, [N
       value = value.Replace("\r", "");
 
       var clearTextBoxWithoutTriggeringPostBack = Keys.Control + "a" + Keys.Control + Keys.Delete;
-      value = clearTextBoxWithoutTriggeringPostBack + value;
 
-      s_log.DebugFormat("FillInWith without triggering PostBack on clear: '{0}'.", value);
+      //GeckoDriver cannot handle the delete command and inputting the value in the same SendKeys after Firefox v99 (would lead to empty field).
+      if (scope.Browser == Browser.Firefox)
+      {
+        s_log.DebugFormat("FillInWith without triggering PostBack on clear: '{0}'.", clearTextBoxWithoutTriggeringPostBack);
+        s_log.DebugFormat("FillInWith without triggering PostBack on inputting value: '{0}'.", value);
```

Comment:
I think we should combine this into one message. 
"FillInWith without triggering PostBack on clear: '{0}' followed by '{1}'."

Reasoning: we have two operations that are meant to work together and you need to either move the log-statements interleaved with the sendkey operations and update the text to be a bit more aware of the split operation or we just make it one message and acknowldge that this is just for firefix